packages/analytics-react-native/src/plugins/context.ts

```diff
@@ -81,7 +81,7 @@ export class Context implements BeforePlugin {
       device_id: this.config.deviceId,
       session_id: this.config.sessionId,
       time,
-      ...(this.config.appVersion && { app_version: appVersion }),
+      ...(appVersion && { app_version: appVersion }),
```

Comment:
Thanks @bgiori, this change makes sense.

Noticed during review, do you know why the Native Module overrides the `config` values. I would expect any explicit config values to take precedence.

Reply:
Hmm, not sure, though I'd be the one to ask since I wrote it. 

I think that may have been an oversight on my part.

Reply:
I agree that explicit config should take precedence over system values.

Reply:
👍 I'll make a new ticket for that fix

Reply:
FWIW analytics-browser does the same, we use uaresult values not the config values

Reply:
https://amplitude.atlassian.net/browse/AMP-81082